### DISCUSSION TITLE
Не резолвятся домены после обновления 3.6.4

### DISCUSSION THREAD

**Discussion by tunivi** [xtrime-ru/antizapret-vpn-docker#115] [Q&A]
Обновился до последнего релиза, и DNS-сервер в контейнере перестал резолвить домены. При этом внутренний адрес DNS 10.224.0.1 пингуется. Антизапрет использую через Amnezia. Проверял на роутерах Keenetic и Android-телефонах.

**Comment by xtrime-ru** [comment]
В чем была загвоздка?

**Reply by Murakumo-JP** [reply]
> В чем была загвоздка?

После обновы на последнею версию тоже был глюк что не резолвились сайты, оказалось что отвалилась обнова списков запрета пришлось в ENV переменную добавить SKIP_UPDATE_FROM_ZAPRET=false и после пересобрать, после чего обновились списки и всё заработала, после удаления переменой и снова пересборки всё заработала штатно.
